src/helpers/inputs.ts

```diff
@@ -0,0 +1,15 @@
+import { CheckboxItem } from "../components/Checkboxes";
+import { RadioButton } from "../components/Radios";
+
+export const getInputId = (
+  input: CheckboxItem | RadioButton,
```

Comment:
```suggestion
  input: { id?: string, value: string },
```

You only care about a subset of the values of those items, so doing this makes it more flexible.

Reply:
Nice, thank you